src/client.js

```diff
@@ -36,25 +27,26 @@ export default class SleuthClient {
      * @param {Object} params
      */
     async _get(endpoint, params) {
-        try {
-            const uri = this.url + endpoint + SleuthClient._stringParams(params);
-            const response = await fetch(uri);
-            return await SleuthClient._json(response);
-        } catch (ex) {
-            throw ex;
-        }
+        const uri = this.url + endpoint + SleuthClient._stringParams(params);
+        return new Promise(function(resolve, reject) {
+            fetch(uri).then(response => response.json())
+            .then(response => resolve(response))
+            .catch(error => {
+                console.trace(error)
+                reject(error)
+            });
+        })
```

Comment:
Are you sure this is necessary? If you leave `_json()` and this function exactly the same as they were what happens?

Reply:
Were you, by any chance, trying to do 
```javascript
try {
    const uri = this.url + endpoint + SleuthClient._stringParams(params);
    return await SleuthClient._json(fetch(uri));
} catch (ex) {
    throw ex;
}
```

or something like that?

Reply:
Oh huh, the old version works fine. Should I revert it?